compiler/rustc_infer/src/infer/nll_relate/mod.rs

```diff
@@ -93,7 +93,7 @@ pub trait TypeRelatingDelegate<'tcx> {
     );
 
     fn const_equate(&mut self, a: ty::Const<'tcx>, b: ty::Const<'tcx>);
```

Comment:
can you change `const_equate` to also use `register_obligations`?

Reply:
It doesn't actually register any obligations... I removed it